apps/ui/src/components/SpaceDelegates.vue

```diff
@@ -145,47 +161,122 @@ watchEffect(() => setTitle(`Delegates - ${props.space.name}`));
           <div
             v-for="(delegate, i) in delegates"
             :key="i"
-            class="border-b flex space-x-1"
+            class="border-b flex space-x-3 px-4"
           >
-            <div class="flex items-center w-[60%] pl-4 py-3 gap-x-3 truncate">
-              <UiStamp :id="delegate.user" :size="32" />
-              <router-link
-                :to="{
-                  name: 'space-user-statement',
-                  params: {
-                    id: `${space.network}:${space.id}`,
-                    user: delegate.user
-                  }
-                }"
-                class="overflow-hidden leading-[22px]"
+            <router-link
+              :to="{
+                name: 'space-user-statement',
+                params: {
+                  id: `${space.network}:${space.id}`,
+                  user: delegate.user
+                }
+              }"
+              class="flex w-full"
+            >
+              <div
+                class="flex grow sm:grow-0 sm:shrink-0 items-center w-[190px] py-3 gap-x-3 leading-[22px] truncate"
+              >
+                <UiStamp :id="delegate.user" :size="32" />
+                <div class="flex flex-col truncate">
+                  <h4
+                    class="text-skin-link truncate"
+                    v-text="delegate.name || shorten(delegate.user)"
+                  />
+                  <div
+                    class="text-[17px] text-skin-text truncate"
+                    v-text="shorten(delegate.user)"
+                  />
+                </div>
+              </div>
+              <div
+                class="hidden sm:flex items-center grow text-[17px] overflow-hidden leading-[22px] text-skin-heading"
+              >
+                <div
+                  v-if="delegate.statement"
+                  class="clamped-text"
+                  v-text="
+                    removeMarkdown(delegate.statement.statement).slice(0, 130)
```

Comment:
This `slice` is a bit abrupt. Maybe we should use `shorten(val, 130)` instead so we get `...` when needed?

Right now it just snaps in weird way.
<img width="1447" alt="image" src="https://github.com/user-attachments/assets/77fa9447-cb3c-4a7a-9c56-6256b9b07de0">

Also maybe we should increase `130` to bigger limit so on typical computer screen it can actually break 2 line-clamp.

Reply:
Let's use both method. Not sure how large is a typical computer screen, just bumped the limit to 250 chars